cohttp-lwt-unix/test/dune

```diff
@@ -41,3 +41,14 @@
  (package cohttp-lwt-unix)
  (action
   (run ./test_body.exe)))
+
+(executable
+ (modules test_client)
+ (name test_client)
+ (libraries cohttp_lwt_unix_test cohttp-lwt-unix))
+
+(rule
+ (alias runtest)
+ (package cohttp-lwt-unix)
+ (action
+  (run ./test_client.exe)))
```

Comment:
You can rewrite this in a single stanza:

```
(test
 (name test_client)
 (modules test_client)
 (package ..)
 (libraries ...))
```

Reply:
That’s some legacy idiom, I’d say let’s leave it as is, and change it in a separate PR later